### PR TITLE
Spelling fixes for editors guide

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: docs/editor_manual/documents_images_snippets/snippets.rst [hunk 1]
```diff
@@ -6,6 +6,6 @@ Snippets
 
 .. UNSURE HOW TO WRITE THIS AS THE ADVERT EXAMPLE IN WAGTAIL DEMO IS NOT A PARTICULARLY HELPFUL USE CASE. 
 
-When creating a page on a website, it is a common occurance to want to add in a piece of content that already exists on another page. An example of this would be a person's contact details, or an advert that you want to simply show on every page of your site, without having to manually apply it.
+When creating a page on a website, it is a common occurrence to want to add in a piece of content that already exists on another page. An example of this would be a person's contact details, or an advert that you want to simply show on every page of your site, without having to manually apply it.
```
Comment (tomdyson) [inline docs/editor_manual/documents_images_snippets/snippets.rst]: to simply show -> to show

File: docs/editor_manual/finding_your_way_around/the_dashboard.rst [hunk 2]
```diff
@@ -14,7 +14,7 @@ You can return to the Dashboard at any time by clicking the Wagtail log in the t
 .. image:: ../../_static/images/screen02_dashboard_editor.png
 
 - Clicking the logo returns you to your Dashboard.
-- Thes stats at the top of the page describe the total amount of content on the CMS (just for fun!).
+- The stats at the top of the page describe the total amount of content on the CMS (just for fun!).
```
Comment (tomdyson) [inline docs/editor_manual/finding_your_way_around/the_dashboard.rst]: drop "(just for fun!)"

File: docs/editor_manual/new_pages/previewing_and_submitting_for_moderation.rst [hunk 3]
```diff
@@ -3,7 +3,7 @@ Previewing and submitting pages for moderation
 
 The Save/Preview/Submit for moderation menu is always present at the bottom of the page edit/creation screen. The menu allows you to perform the following actions, dependent on whether you are an editor, moderator or administrator:
 
-* **Save draft:** Saves your current changes but doesn’t submit the page for moderation and so won’t be published. (all roles)
+* **Save draft:** Saves your current changes but doesn't submit the page for moderation and so won’t be published. (all roles)
```
Comment (tomdyson) [inline docs/editor_manual/new_pages/previewing_and_submitting_for_moderation.rst]: won’t -> won't (apostrophe)

File: docs/editor_manual/new_pages/selecting_a_page_type.rst [hunk 4]
```diff
@@ -8,5 +8,5 @@ Selecting a page type
 
 .. image:: ../../_static/images/screen10_blank_page_edit_screen.png
 
-* Once you’ve selected a page type you will be presented with a blank New page screen.
+* Once you've selected a page type you will be presented with a blank New page screen.
```
Comment (tomdyson) [inline docs/editor_manual/new_pages/selecting_a_page_type.rst]: New page -> New Page

File: docs/editor_manual/new_pages/inserting_images.rst [hunk 5]
```diff
@@ -58,12 +58,12 @@ ___________________________________
 
 Images can also be inserted into the body text of a page. When editing the Body field of a standard page, click the image illustrated above. You will then be presented with the same options as for inserting images into the main carousel.
 
-In addition, the Wagtail Demo site allows you to chose an aligmnent for you image.
+In addition, the Wagtail Demo site allows you to chose an alignment for you image.
```
Comment (tomdyson) [inline docs/editor_manual/new_pages/inserting_images.rst]: you image -> your image

**Comment by kaedroho** [comment]
Rebased

**Pull request merged**